rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/OneScanHelper.java

```diff
@@ -45,7 +45,8 @@ private void send(String action, Bundle extras, IntentResultCallback callback) {
         if (!isCallable(mActivity, intent)) {
             Log.e(TAG, "Intent is not Callable!");
             if (callback != null) {
-                callback.onResult(Activity.RESULT_CANCELED, null);
+                extras.putBoolean(Constants.Result.ONESCAN_IS_NOT_INSTALLED, true);
```

Comment:
Also add a test to verify that the callback here is called with the correct extras when onescan is not installed.

Reply:
I have tried playing around with `OneScanHelperTest.sendShouldVerifyScenarioFirst()`, but I can't get the updated extra. It means that I cant validate.

Anyway, before I reuse that `extras` variable, I use new instance of Bundle as argument. Then when I try to test, the instance always null. @syedowaisali90 knows about this. Maybe we have problem with the test environment?

Reply:
Done, @syedowaisali90 shows me the simplest way.